README.md

```diff
@@ -288,7 +288,7 @@ PRs welcomed!
 
 ### Physics
 
-- [physics](https://github.com/Leedehai/typst-physics) - A library for usual physics notations, e.g. vectors and vector fields, matrices, differentials, derivatives, Dirac brackets, tensors, isotopes, and digital signal sequences.
+- [physics](https://github.com/Leedehai/typst-physics) - A library for usual physics notations, e.g. vectors and vector fields, matrixes, differentials, derivatives, Dirac brackets, tensors, isotopes, and digital signal sequences.
```

Comment:
Both "matrices" and "matrixes" are [correct][1] plurals for "matrix". No change needed here.

[1]: https://www.merriam-webster.com/dictionary/matrix